### PR TITLE
kmp: update versions for 0.6.0 release

### PR DESCRIPTION
**Pull request by buenaflor**
Update versions for 0.6.0 release

### CODE REVIEW

File: platform-includes/getting-started-primer/kotlin-multiplatform.mdx [hunk 1]
```diff
@@ -18,6 +18,7 @@ Use the Kotlin Multiplatform and Cocoa SDK combinations listed in the table belo
 | Up to 0.3.0                      | 8.4.0             |
 | 0.4.0                            | 8.17.2            |
 | 0.5.0                            | 8.21.0            |
+| 0.6.0                            | 8.25.0            |
 
 <Note>
 Starting May 1, 2024, Apple requires all apps submitted to the App Store to provide a list of privacy-related APIs they use. Use Sentry Cocoa SDK version `8.21.0` to ensure compliance with these requirements.
```
Comment (kahest) [inline platform-includes/getting-started-primer/kotlin-multiplatform.mdx:24]:
```suggestion
| 0.6.0                            | 8.25.0            |

<Note>
Starting May 1, 2024, Apple requires all apps submitted to the App Store to provide a list of privacy-related APIs they use. Use Sentry Cocoa SDK version `8.25.0` to ensure compliance with these requirements.
```

File: docs/platforms/kotlin-multiplatform/native-access-sdk.mdx [hunk 2]
```diff
@@ -27,7 +27,10 @@ cocoapods {
   ios.deploymentTarget = "14.1"
   podfile = project.file("../iosApp/Podfile")
   // Make sure you use the proper version according to our Cocoa SDK Version Compatibility Table.
-  pod("Sentry", "~> 8.20.0")
+  pod("Sentry") {
+    version = "8.25.0"
```
Comment (kahest) [inline docs/platforms/kotlin-multiplatform/native-access-sdk.mdx:31]: `m`: do we want to lock this to 8.25.0 or <9.0.0?

**Review by kahest — COMMENTED** [review]
thanks for updating! left clarifying questions

**Review by vivianyentran — APPROVED** [review]

File: docs/platforms/kotlin-multiplatform/native-access-sdk.mdx (hunk 2 above)
Comment (buenaflor) [reply docs/platforms/kotlin-multiplatform/native-access-sdk.mdx:31]: We could do `~> 8.25` I think then we wouldn't always need to update these code snippets

Comment (kahest) [reply docs/platforms/kotlin-multiplatform/native-access-sdk.mdx:31]: sounds reasonable to me

**Pull request merged**